### PR TITLE
ADR Allow download of released Interactive reports

### PR DESCRIPTION
**Pull request by lucyb**
Fixes: #2669

### CODE REVIEW

File: docs/adr/0009-download-released-reports.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# 1. Allow download of released reports
```
Comment (ghickman) [inline docs/adr/0009-download-released-reports.md:1]: Should this be `9.`?

**Review by ghickman — APPROVED** [review]
Looks great 👍

Can we use semantic line breaks at the end of sentences?

File: docs/adr/0009-download-released-reports.md (hunk 1 above)
Comment (lucyb) [reply docs/adr/0009-download-released-reports.md]: Yes. *sigh*

**Comment by lucyb** [comment]
> Can we use semantic line breaks at the end of sentences?

This isn't something I've used previously. I can guess what they are from your ADR8. Why are they used and is it something we've used elsewhere? I find it harder to read the markdown when they there.

**Pull request merged**